packages/core/helper-plugin/lib/src/components/RelationInput/RelationInput.js

```diff
@@ -46,7 +48,7 @@ export const RelationInput = ({
             <FieldLabel>{label}</FieldLabel>
             <ReactSelect
               components={{ Option }}
-              options={searchResults}
+              options={data?.pages || []}
```

Comment:
```suggestion
              options={searchResults.pages.flat()}
```

`.pages` with always be defined and may be an empty array.

Reply:
mmh the structure is still `searchResults.data.pages` right?
curious about the flat, will we need it?

Reply:
Yes, we will need it, because pages looks like:

```js
pages: [
  [ { id: 1}, { id: 2 } ]
]
```

Reply:
oh but then PropTypes aren't good there? 🤔

<img width="593" alt="image" src="https://user-images.githubusercontent.com/71838159/186436106-c0d66d79-b652-490f-84d4-bec491dfa4de.png">

Reply:
True. They need to be updated for both (`relations` and `search`). Sorry for that - my bad.

Reply:
on it :)

Reply:
fixed fixed [here](https://github.com/strapi/strapi/pull/14189/commits/54c41cf84006dac402f4872e188a305460103f26)